pagination/helpers.go

```diff
@@ -0,0 +1,54 @@
+package pagination
+
+import (
+	"encoding"
+	"encoding/base64"
+)
+
+// PageSizeGetter holds a method to return the amount of pages requested by users when listing items in an API call.
+type PageSizeGetter interface {
+	// GetPageSize returns the desired page size.
+	GetPageSize() int32
+}
+
+// PageSize returns a valid page size following the AIP-158 for Pagination.
+// Reference: https://google.aip.dev/158
+//
+//	Default value: 50
+//	Max value: 1000
```

Comment:
I'd prefer if we don't change the max values, I already mentioned this in another PR, but I believe we should stick to the guidelines, instead of worrying about performance as it is right now.

Reply:
That's fair, let's go with `100` then.